src/extension/ip_extraction.c

```diff
@@ -424,7 +425,9 @@ static bool _parse_ip_address(
     if (ret != 1) {
         ret = inet_pton(AF_INET6, addr, &out->v6);
         if (ret != 1) {
-            mlog(dd_log_info, "Not recognized as IP address: \"%s\"", addr);
+            if (ip_or_error) {
```

Comment:
Remind me why we need this?

Reply:
Yes. So `via` header can contains ips or not
```
Via: [ <protocol-name> "/" ] <protocol-version> <host> [ ":" <port> ]
Via: [ <protocol-name> "/" ] <protocol-version> <pseudonym>
```
When it comes with a pseudonym, it was throwing errors to the logs this way avoid that happening on that header

Reply:
Right but the log level here is on info so we don't need to prevent it, the tracer is a bit less flexible in that respect.

Reply:
Ok, I will change that